sqldelight-gradle/android-plugin/src/test/kotlin/com/squareup/sqldelight/GenerateSchemaTest.kt

```diff
@@ -0,0 +1,28 @@
+package com.squareup.sqldelight
+
+import com.google.common.truth.Truth
+import org.gradle.testkit.runner.GradleRunner
+import org.junit.Test
+import java.io.File
+
+class GenerateSchemaTest {
+  @Test fun `schema file generates correctly`() {
+    val fixtureRoot = File("src/test/schema-file")
+
+    GradleRunner.create()
+        .withProjectDir(fixtureRoot)
+        .withPluginClasspath()
+        .withArguments(
+            "clean", "generateSqlDelightSchema", "--stacktrace",
+            "-Dsqldelight.skip.runtime=true"
+        )
+        .build()
+
+    // verify
+    val schemaFile = File(fixtureRoot, "src/main/sqldelight/databases/1.db")
+    Truth.assertThat(schemaFile.exists())
+        .isTrue()
+
+    schemaFile.delete()
```

Comment:
If the test fails this file will be left on the file system. Maybe remove this and add a .gitignore in the fixture folder for the generated file.